src/psyclone/domain/lfric/kern_call_acc_arg_list.py

```diff
@@ -113,6 +113,60 @@ def stencil(self, arg, var_accesses=None):
         var_name = DynStencils.dofmap_name(self._kern.root.symbol_table, arg)
         self.append(var_name, var_accesses)
 
+    def stencil_2d(self, arg, var_accesses=None):
+        '''Add 2D general stencil information associated with the argument 'arg'
+        to the argument list. OpenACC requires the full dofmap to be
+        specified. If supplied it also stores this access in var_accesses.
+
+        :param arg: the meta-data description of the kernel \
+            argument with which the stencil is associated.
+        :type arg: :py:class:`psyclone.dynamo0p3.DynKernelArgument`
+        :param var_accesses: optional VariablesAccessInfo instance to store \
+            the information about variable accesses.
+        :type var_accesses: \
+            :py:class:`psyclone.core.access_info.VariablesAccessInfo`
+
+        '''
+        from psyclone.dynamo0p3 import DynStencils
```

Comment:
Since this implementation is identical (I think?) to that for `stencil()`, I'd just call `stencil()` here with a note to that effect.

Reply:
Done.